gallery/src/components/NxVulnerabilityIndicatorLegend/NxVulnerabilityIndicatorLegendPage.tsx

```diff
@@ -0,0 +1,150 @@
+import NxVulnerabilityIndicatorLegendVerticalExample from './NxVulnerabilityIndicatorLegendVerticalExample';
+import NxVulnerabilityIndicatorLegendVerticalExampleHalf from './NxVulnerabilityIndicatorLegendVerticalExampleHalf';
+
+const NxVulnerabilityIndicatorLegendHorizontalExampleCode = require(
+        './NxVulnerabilityIndicatorLegendHorizontalExample?raw'
+    ),
+    NxVulnerabilityIndicatorLegendHorizontalExampleHalfCode = require(
+        './NxVulnerabilityIndicatorLegendHorizontalExampleHalf?raw'
+    ),
+    NxVulnerabilityIndicatorLegendVerticalExampleCode = require(
+        './NxVulnerabilityIndicatorLegendVerticalExample?raw'
+    ),
+    NxVulnerabilityIndicatorLegendVerticalExampleHalfCode = require(
+        './NxVulnerabilityIndicatorLegendVerticalExampleHalf?raw'
+    );
+
+const NxVulnerabilityIndicatorLegendPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        <NxCode>NxVulnerabilityIndicatorLegend</NxCode> is an element to display the legend for
+        various vulnerability ratings. The legend can be in two formats: horizontal and vertical.
+      </NxP>
+      <NxInfoAlert>
+        By default, <NxCode>NxVulnerabilityIndicatorLegend</NxCode> renders a horizontal legend.
+        To render a vertical legend, the prop <NxCode>vertical</NxCode> needs to be provided.
+      </NxInfoAlert>
+      <NxP>
+        <NxCode>NxVulnerabilityIndicatorLegend</NxCode> supports displaying the legend for
+        'unspecified', 'none', 'low', 'moderate', 'severe', and 'critical' vulnerability ratings.
+      </NxP>
+      <NxTable>
+        <NxTable.Head>
+          <NxTable.Row>
+            <NxTable.Cell>Prop</NxTable.Cell>
+            <NxTable.Cell>Type</NxTable.Cell>
+            <NxTable.Cell>Required</NxTable.Cell>
+            <NxTable.Cell>Default</NxTable.Cell>
+            <NxTable.Cell>Details</NxTable.Cell>
+          </NxTable.Row>
+        </NxTable.Head>
+        <NxTable.Body>
+          <NxTable.Row>
+            <NxTable.Cell>header</NxTable.Cell>
+            <NxTable.Cell>string</NxTable.Cell>
+            <NxTable.Cell>no</NxTable.Cell>
+            <NxTable.Cell>'Legend'</NxTable.Cell>
+            <NxTable.Cell>Sets the legend header title.</NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell>vertical</NxTable.Cell>
+            <NxTable.Cell>boolean</NxTable.Cell>
+            <NxTable.Cell>no</NxTable.Cell>
+            <NxTable.Cell>false</NxTable.Cell>
+            <NxTable.Cell>Displays a vertical legend.</NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell>critical</NxTable.Cell>
+            <NxTable.Cell>boolean</NxTable.Cell>
+            <NxTable.Cell>no</NxTable.Cell>
+            <NxTable.Cell>false</NxTable.Cell>
+            <NxTable.Cell>Displays the legend item for critical vulnerability items.</NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
```

Comment:
A bunch of these still have the policy threat levels, and need to be updated

Reply:
Updated in: [cc2bfac](https://github.com/sonatype/sonatype-react-shared-components/pull/1199/commits/cc2bfac71027ac309f17c387261135fb65f7eedd)